site.conf

```diff
@@ -15,8 +15,8 @@
 # Pd, iOS, OpenFrameworks, etc.  See "README.embed" for more information.
 #BUILDTYPE = STANDALONE
 #BUILDTYPE = OSXEMBEDDED
-BUILDTYPE = MAXMSP
-#BUILDTYPE = PD
+#BUILDTYPE = MAXMSP
+BUILDTYPE = PD
```

Comment:
Sorry, it was set up for my builds. Shouldn't the default buildtype be standalone?